reference/ats/openapi.yaml

```diff
@@ -0,0 +1,671 @@
+openapi: 3.0.0
+x-stoplight:
+  id: 79ufakk1ygzmg
+info:
+  title: ATS API Reference
+  version: '2020-09-17'
+  description: |-
+    The Finch ATS (Applicant Tracking System) API provides a unified way to connect to several ATS systems. The API requires an access token issued by Finch.
```

Comment:
To match the [HRIS page](https://tryfinch.stoplight.io/docs/reference/branches/ar%2Fats/0c7c919255262-hris-api-reference), I'd nitpick the wording here a bit or update the wording on the HRIS page.

The HRIS page is a little strange right now because where it says "Please see [here](https://developer.tryfinch.com/docs/reference/branches/open-api/docs/%20Finch%20API/1%20-%20Getting%20Started.md) for an overview of the API", when you click on the "here" it takes you to a "What is Finch?" page. Now that we have an ATS product, I think this needs updating.